src/frontend/src/components/authenticateBox.ts

```diff
@@ -649,3 +588,120 @@ const pinIdentityToDerPubkey = async (
     pinIdentity.publicKey
   )) as DerEncodedPublicKey;
 };
+
+// Find and use a passkey, whether PIN or webauthn
+const usePasskeyFlow = async <T, I>({
```

Comment:
I'm confused by the  naming and the comment. Passkey refers to WebAuthn only, so having this method cover PIN as well is unintuitive for me.

Maybe `authnFlow`?

Reply:
> Passkey refers to WebAuthn only

No, we call our "devices" passkeys too...

> Maybe authnFlow?

We have a few of these already (authenticate, authenticateBox, etc), I wanted to be more specific. `useIdentity`? `useDevice`?

Reply:
>> Passkey refers to WebAuthn only
>
> No, we call our "devices" passkeys too...

We don't call the PIN auth passkey, we call that temp key 🙃 

Either way `useIdentity` is fine for me 👍

Reply:
Done!